neo/Network/P2P/Peer.cs

```diff
@@ -98,6 +100,8 @@ private static bool IsIntranetAddress(IPAddress address)
 
         protected abstract void NeedMorePeers(int count);
 
+        public abstract NetworkAddressWithTime[] GetRandomConnectedPeers(int count);
```

Comment:
`NetworkAddressWithTime` shouldn't appear in the class `Peer`.

Reply:
fixed